.travis.yml

```diff
@@ -3,8 +3,8 @@ cache: npm
 notifications:
   email: false
 node_js:
-  - 10.0.0
-  - 12
+  - lts/dubnium
```

Comment:
```suggestion
  - 10
```

Reply:
@MichaelDeBoey why do you want to change that?

Reply:
The latest Node 10 version will be used.

This will make the change in this PR as minimal as possible

Reply:
Yeah, I'm pretty sure that all that needs to happen for this is to change `10.0.0` to `10` and we'll be good. The release names are a little confusing and need a reference lookup anytime we want to check them 😬

Reply:
Ok. Committed. I always use the release names with "lts/" to remind people not to change it to the new non-lts release whenever it comes out.